AmahiAnywhere/AmahiAnywhere/ConnectionTableViewController.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  ConnectionTableViewController.swift
+//  AmahiAnywhere
+//
+//  Created by Syed on 21/02/18.
+//  Copyright © 2018 Amahi. All rights reserved.
+//
+
+import UIKit
+
+class ConnectionTableViewController: UITableViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        // Uncomment the following line to preserve selection between presentations
+        // self.clearsSelectionOnViewWillAppear = false
+
+        // Uncomment the following line to display an Edit button in the navigation bar for this view controller.
+        // self.navigationItem.rightBarButtonItem = self.editButtonItem
+    }
+
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
+        tableView.backgroundColor = UIColor.black
+    }
+
+    // MARK: - Table view data source
+    override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        print(indexPath.row)
```

Comment:
You could possibly have used breakpoints for debugging instead of using `print` statements like these.
Also, if `print` is required for another purpose, kindly prefer using `debugPrint'